src/compiler/commandLineParser.ts

```diff
@@ -996,9 +996,9 @@ namespace ts {
         return { options, errors };
     }
 
-    export function convertTypingOptionsFromJson(jsonOptions: any, basePath: string, configFileName?: string): { options: TypingOptions, errors: Diagnostic[] } {
+    export function convertTypeAcquisitionFromJson(jsonOptions: any, basePath: string, configFileName?: string): { options: TypeAcquisition, errors: Diagnostic[] } {

```

Comment:
we never documented it intentionally , i would say. so it is a breaking change, but not one that would impact ppl.